### PR TITLE
Paella player playbar optimisations

### PR DESCRIPTION
**Pull request by snoesberger**
Optimizes the behavior and design of the paella player:
- Adds transparency to the paella playbar
- Playbar hides after 1 instead of 5 sec.
- Black as background color for paella player video container

### CODE REVIEW

**Comment by dagraf** [comment]
We tested this optimizations and they work as expected.

@ferishili: Can you please review this PR and merge it, if everything is fine or comment it, if not. Thx!

File: js/opencast/src/Paella/config/config.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
 
   "defaultAudioStream": "presenter",
 
+  "ui": {
+            "hideUITimer": 1000,
```
Comment (ferishili) [inline js/opencast/src/Paella/config/config.json]: Indentation!
```suggestion
    "hideUITimer": 1000,
```

File: js/opencast/src/Paella/config/config.json [hunk 2]
```diff
@@ -11,6 +11,11 @@
...
+            "hideOnMouseLeave": true
```
Comment (ferishili) [inline js/opencast/src/Paella/config/config.json]: Indentation also here!
```suggestion
    "hideOnMouseLeave": true
```

**Review by ferishili — COMMENTED** [review]
Thanks for your work, the changes look fine and provide UX improvements.

**Comment by dagraf** [comment]
@ferishili: Do you not have the necessary rights to merge this PR? Or is there another reason why you did not merge it right away but labeled it as "Ready to merge"?

**Comment by ferishili** [comment]
@dagraf thanks for asking, I have the right, however, since Fabian does the versioning and wanted to also keep track of the new changes and add them into `release_9` PR, I thought that he would take care of the merging of the PRs. As you may see, PRs from me or reviewed by me are mostly closed, not merged only because of this matter.

**Comment by chfsx** [comment]
This PR has been merged locally and is part of https://github.com/opencast-ilias/OpenCast/releases/tag/v8.3.0

**Pull request closed without merging**